rust_src/src/frames.rs

```diff
@@ -44,7 +44,7 @@ impl LispFrameRef {
             match self.vertical_scroll_bar_type() {
                 vertical_scroll_bar_type::vertical_scroll_bar_left
                 | vertical_scroll_bar_type::vertical_scroll_bar_right => {
-                    return self.config_scroll_bar_width
+                    return self.config_scroll_bar_width;
                 }
                 _ => return 0,
```

Comment:
The return statements here are redundant, you can remove them.

Reply:
Disregard what I just wrote, I'd actually suggest removing the added semicolon and leaving this be for now, as there's another open PR that fixes this.